src/internal/storage/renew/string_set.go

```diff
@@ -71,6 +71,17 @@ func (ss *StringSet) Add(ctx context.Context, x string) error {
 	return nil
 }
 
+// Compose returns the result of a compose call on all of the strings being renewed.
+func (ss *StringSet) Compose(ctx context.Context) (string, error) {
```

Comment:
This doesn't affect the internal state of the StringSet like the background composing loop does.  What is this for?

Reply:
Right, this is just to get a single file set that references all of the file sets being renewed. This is useful for keeping the file sets being renewed alive across processes. The current use case is that we have a task that creates all of the file sets for the datum sets and we want the worker master to be able to keep those alive while the job is being processed (see `CreateDatumSetsTask`)